src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -68,8 +67,10 @@ public class Swerve extends SubsystemBase { // physicalproperties/conversionFact
     private final PPHolonomicDriveController pathPlannerController = new PPHolonomicDriveController(SwerveK.translationConstants, SwerveK.rotationConstants);
     private final NetworkTable table = NetworkTableInstance.getDefault().getTable("Robot").getSubTable("swerve");
     private boolean initializedOdometryFromVision = false;
+    private BooleanSupplier overridePathPlanner = () -> false;
```

Comment:
```suggestion
    private final BooleanSupplier overridePathPlanner;
```
Expresses intent better imo